app/models/vim_performance_tag_value.rb

```diff
@@ -54,20 +54,14 @@ def self.build_for_association(parent_perf, assoc, options = {:save => true})
     ts = parent_perf.timestamp
     children = parent_perf.resource.send("#{assoc}_from_vim_performance_state_for_ts", ts)
     return [] if children.empty?
+    vim_performance_daily = parent_perf.kind_of?(VimPerformanceDaily)
+    recs = get_metrics(children, ts, parent_perf.capture_interval_name, vim_performance_daily, options[:category])
```

Comment:
By not passing in a rich object, I am trying to make `get_metrics` behavior more well-defined(confined) and easier to test.